packages/postcss-discard-unused/src/index.js

```diff
@@ -54,7 +54,9 @@ function hasFont(fontFamily, cache, comma) {
 function filterFont({ atRules, values }, comma) {
   values = [...new Set(values)];
   atRules.forEach((r) => {
-    const families = r.nodes.filter(({ prop }) => prop === 'font-family');
+    const families = r.nodes.filter(
+      (node) => node.type === 'decl' && node.prop === 'font-family'
```

Comment:
Same issue, `nodes` is of type `ChildNode`, so we need to check whether it's a delcaration, even if in this case we know it's the child of a Rule.